### PR TITLE
Add new profile windows-executable

### PR DESCRIPTION
**Pull request by nik-sta**
Add new profile windows-executable for building .exe packages direct
from maven.

### CODE REVIEW

File: pom/pom.xml [hunk 1]
```diff
@@ -85,10 +85,24 @@
         <!-- Default minify settings -->
         <minify.skip>false</minify.skip>
 
+        <!-- Executable Main class -->
+        <executable.main.class></executable.main.class>
+        <executable.finalName>executable-output</executable.finalName>
+
         <!-- Default executable jar main class -->
-        <executable.jar.main.class></executable.jar.main.class>
         <executable.jar.finalName>executable-output</executable.jar.finalName>
```
Comment (itsmefox) [inline pom/pom.xml]: Use here also ${executable.finalName} or delete if not needed

File: pom/pom.xml [hunk 2]
```diff
@@ -207,6 +221,84 @@
...
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>3.6.0</version>
+                    <configuration>
+                        <source>${jdkLevel}</source>
+                        <target>${jdkLevel}</target>
+                    </configuration>
+                </plugin>
```
Comment (itsmefox) [inline pom/pom.xml]: Why we have to redefine `maven-compiler-plugin`? Its is already defined in `build / plugins`

File: pom/pom.xml [hunk 3]
```diff
@@ -207,6 +221,84 @@
...
+                            <id>l4j-clui</id>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>launch4j</goal>
+                            </goals>
+                            <configuration>
+                                <headerType>gui</headerType>
+                                <jar>${project.build.directory}/${artifactId}-${version}-shaded.jar</jar>
+                                <outfile>${project.build.directory}/${executable.finalName}.exe</outfile>
+                                <downloadUrl>http://java.com/download</downloadUrl>
```
Comment (itsmefox) [inline pom/pom.xml:264]: What is the purpose of this property (`<downloadUrl>http://java.com/download</downloadUrl>`)?

File: pom/pom.xml [hunk 4]
```diff
@@ -85,10 +85,24 @@
...
+        <!-- Executable Main class -->
+        <executable.main.class></executable.main.class>
+        <executable.finalName>executable-output</executable.finalName>
+
         <!-- Default executable jar main class -->
-        <executable.jar.main.class></executable.jar.main.class>
         <executable.jar.finalName>executable-output</executable.jar.finalName>
         <executable.jar.appendAssemblyId>true</executable.jar.appendAssemblyId>
+
+        <!-- Windows executable -->
```
Comment (itsmefox) [inline pom/pom.xml]: Rename to `<!-- Default Windows executable settings -->`

File: pom/pom.xml [hunk 5]
```diff
@@ -85,10 +85,24 @@
         <!-- Default minify settings -->
         <minify.skip>false</minify.skip>
 
+        <!-- Executable Main class -->
+        <executable.main.class></executable.main.class>
+        <executable.finalName>executable-output</executable.finalName>
+
         <!-- Default executable jar main class -->
```
Comment (itsmefox) [inline pom/pom.xml]: Rename to `<!-- Default Jar executable settings -->`

**Review by itsmefox — CHANGES_REQUESTED** [review]

File: pom/pom.xml (hunk 3 above)
Comment (nik-sta) [reply pom/pom.xml:264]: To download if not installed the JRE.

File: pom/pom.xml [hunk 6]
```diff
@@ -227,11 +226,6 @@
                 <plugin>
```
Comment (itsmefox) [inline pom/pom.xml:226]: This plugin (`maven-compiler-plugin`) is already global defined in `build / plugins` and you don't need to add this again.
See [Line 465](https://github.com/Viascom/groundwork/blob/7fd66d0b3680915a2f7daee16058e32d96a202b8/pom/pom.xml#L465)

**Review by itsmefox — CHANGES_REQUESTED** [review]

File: pom/pom.xml (hunk 6 above)
Comment (nik-sta) [reply pom/pom.xml:226]: ok it's now removed ... my fault

**Review by itsmefox — APPROVED** [review]

**Pull request merged**